cfspopcon/algorithms/geometry.py

```diff
@@ -43,6 +44,7 @@ def run_calc_geometry(
 
     plasma_volume = formulas.calc_plasma_volume(major_radius, inverse_aspect_ratio, areal_elongation)
     surface_area = formulas.calc_plasma_surface_area(major_radius, inverse_aspect_ratio, areal_elongation)
+    poloidal_circumference = formulas.calc_plasma_poloidal_circumference(minor_radius, areal_elongation)
```

Comment:
Would you mind adding the `poloidal_circumference` to the list of return values in the doc string?